lib/active_model/nested_attributes.rb

```diff
@@ -13,94 +13,73 @@ module NestedAttributes
     class_methods do
       # NOTE: Options have NOT yet been implemented! (e.g., `allow_destroy`)
       def accepts_nested_attributes_for(*models, **_options)
-        define_getters_for_nested_models(models)
-        define_setters_for_nested_models(models)
-        allow_reflection_on_nested_models(models)
-        allow_serialization_of_nested_models(models)
-        define_convenience_methods
-        define_validation_methods
+        define_attribute_types_for(models)
+        define_attributes_for(models)
+        define_getters_for(models)
+        define_validation_for(models)
+        allow_reflection_on(models)
```

Comment:
The bulk of the changes in this commit are in `ActiveModel::NestedAttributes` which now does more heavy lifting so nested forms are more idiomatic.